src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx

```diff
@@ -1,40 +1,101 @@
 import React from "react"
-import { createFragmentContainer, graphql } from "react-relay"
-import { Header } from "../Header"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+
+import { ArtworkContextAuction_artwork } from "__generated__/ArtworkContextAuction_artwork.graphql"
+import { ArtworkContextAuctionQuery } from "__generated__/ArtworkContextAuctionQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+
 import {
   ArtistArtworkGrid,
   AuctionArtworkGrid,
-  PartnerArtworkGrid,
   RelatedWorksArtworkGrid,
 } from "./ArtworkGrids"
 
-export const ArtworkContextAuction = props => {
+interface ArtworkContextAuctionProps {
+  /** The artworkID to query */
+  artworkID: string
+  /** Used to exclude the current work from the currently-shown work from grid */
+  artworkMongoID: string
+  /** If the artwork  */
```

Comment:
Q- How do you keep a programmer in suspense?
A- Read the comment on Line 20 in this file.

Reply:
lol wonder where my mind went here 😄